azurecontainerapps/src/CommandHelper.ts

```diff
@@ -38,4 +50,39 @@ export class CommandHelper {
             throw err;
         }
     }
+
+    /**
+     * Executes a given command using the pwsh executable.
+     * @param command - the command to execute in PowerShell
+     * @param cwd - the current working directory; if not provided, the 'cwd' input will be used
+     * @returns the string output from the command
+     */
+    private async execPwshCommandAsync(command: string, cwd?: string): Promise<string> {
```

Comment:
Nit: the implementation looks very similar to the `execBashCommandAsync` one. Do you expect that we might want to keep these separated or could we have a common helper method to share most of the code?

Reply:
I thought about trying to consolidate both functions into the `execCommandAsync` function I introduced, but decided to keep them separate _just in case_ issues arise in an Windows or non-Windows agent relating to the corresponding shell we're using and we would need to re-separate the functions to resolve the issue. If you'd like me to try and consolidate the functions, let me know!

Reply:
No OK that seems fair. If we need to add another third implementation sometimes, then it will be a good opportunity to consolidate. Thanks!